dags/vz_cr3_extract_ocr_narrative.py

```diff
@@ -29,35 +29,38 @@
     "HASURA_ADMIN_KEY": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.GraphQL Endpoint key",
-    }, 
+    },
     "AWS_ACCESS_KEY_ID": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.AWS Access key",
     },
     "AWS_SECRET_ACCESS_KEY": {
         "opitem": "Vision Zero CRIS Import",
         "opfield": "production.AWS Secret key",
-    }
+    },
 }
 
+# fmt: off
 with DAG(
     dag_id=f"vz_cr3_ocr_narrative_extract_{DEPLOYMENT_ENVIRONMENT}",
     default_args=default_args,
-    # Every 5 minutes, at 8A, 9A, and 10A
-    schedule_interval="*/5 8-10 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    schedule_interval="*/20 * * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
     tags=["repo:atd-vz-data", "vision-zero"],
     catchup=False,
 ) as dag:
+# fmt: on
```

Comment:
what does this mean?

Reply:
It's a formatting directive to `black` when to turn off and on. `black` wants to break the schedule line up over 3 lines, and I find it much easier to understand with each line being one key/value pair. I really do like black's linting 99.999% of the time, but i wanted to override it here.

Reply:
ah! TIL, thank you!